atlp-rwanda/atlp-devpulse-fn
# Fix tel

**PR Description**
This PR refactors the user model to save the country code alongside the phone number and updates the relevant files accordingly.

**Description of tasks that were expected to be completed**

- Refactor the user model to combine country code and phone number.
- Update files affected by this change (especially those involving user communication).

Previously, the country code and phone number were stored separately in MongoDB, which created a poor user experience for managers and admins when communicating or retrieving a trainee's phone number.

**How has this been tested?**

1. Clone the repo.
2. Checkout the branch fix-tel.
3. Set up the environment variables as indicated in .env.example.
4. Run npm install followed by npm run dev to ensure the app runs successfully.

 **Deployed Link:**

N/A

**Track PR (Issue number & link):**
N/A

Screenshots (If appropriate):
N/A
